airbyte-webapp/src/packages/cloud/views/DefaultView.tsx

```diff
@@ -12,7 +12,7 @@ export const DefaultView: React.FC = () => {
   return (
     <Navigate
       to={
-        workspaces.length > 1
+        workspaces.length !== 1
```

Comment:
@edmundito I pushed this into this PR. When I've tested I noticed, that when removing the permission of a user before they use the link in the mail they are left with a user account but no workspace. That can already happen today, if a user is removed from the last workspace they are in. In this case currently the webapp fails. This makes sure they are landing on the workspace selection page and can create a new workspace.